CMakeLists.txt

```diff
@@ -47,26 +50,37 @@ set(TRITON_BACKEND_REPO_TAG "main" CACHE STRING "Tag for triton-inference-server
 set(TRITON_CORE_REPO_TAG "main" CACHE STRING "Tag for triton-inference-server/core repo")
 set(TRITON_COMMON_REPO_TAG "main" CACHE STRING "Tag for triton-inference-server/common repo")
 
+set(TRITON_ENABLE_GPU OFF CACHE BOOL "GPU disabled" FORCE)
```

Comment:
Do we need this flag?

Reply:
Its needed because some of the included builds from core and common use that flag and we want to make sure it is off for openvino.